### ISSUE TITLE
Dashboard A11y: Settings View - publisher logos should be an identifiable group

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#4203]
## Bug Description

When using a keyboard on the settings view to tab through the page, you have to tab through every publisher logo available. 

## Expected Behaviour

This group of publisher logos should be a semantic list with a clear indicator of what it's of so that content can be skipped if desired. 

## Steps to Reproduce

Tab through the settings page of the dashboard, see that you must tab through all publisher logos, no identifiable group. 

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

- Fix should include karma test

**Comment by csossi** [comment]
Verified in QA
